src/core/pybind.cc

```diff
@@ -87,4 +93,10 @@ PYBIND11_MODULE(polygames, m) {
       .def("set_find_batch_size_max_bs", &ModelManager::setFindBatchSizeMaxBs);
 
   py::class_<SampleResult>(m, "SampleResult").def("get", &SampleResult::get);
+
+
+  // References core::Actor so must go after this file's main defs
```

Comment:
I had a lot of comments to add in this file but they were all answered by the useful comments you left :)